azure-pipelines.yml

```diff
@@ -36,6 +36,14 @@ jobs:
       arguments: '-configuration $(BuildConfiguration) -all -parallel'
       modifyEnvironment: false
 
+  - task: PublishBuildArtifacts@1
```

Comment:
I guess this should not be merged.

Reply:
IDK - sometimes it's helpful to have the artifacts of the failing build, but we should remove it after diagnosing the issue.